src/main/scala/viper/gobra/frontend/Config.scala

```diff
@@ -116,9 +133,38 @@ class ScallopGobraConfig(arguments: Seq[String], isInputOptional: Boolean = fals
     */
   val input: ScallopOption[List[String]] = opt[List[String]](
     name = "input",
-    descr = "List of Gobra programs or a single package name to verify"
+    descr = "List of Gobra programs, a project directory or a single package name to verify",
+    short = 'i'
   )
 
+  val recursive: ScallopOption[Boolean] = toggle(
+    name = "recursive",
+    descrYes = "Verify nested packages recursively",
+    default = Some(false),
+    short = 'r'
+  )
+
+  val includePackages: ScallopOption[List[String]] = opt[List[String]](
+    name = "includePackages",
+    descr = "Which packages should be verified, all packages are verified by default",
+    default = Some(List.empty),
+    short = 'p'
+  )
+
+  val excludePackages: ScallopOption[List[String]] = opt[List[String]](
+    name = "excludePackages",
+    descr = "Which packages should not be verified, all packages are verified by default",
```

Comment:
Would replace with "Packages to ignore. These packages will not be verified, even if they are found in the specified directories."